### PR TITLE
Change to increase job config input size

### PR DESCRIPTION
**Pull request by rmettu-rms**
Passing in 155K job config input string resulted in Akka timeout response to the job.
In WebApi, when creating JobManagerActor it fails to serialize jobConfig object when greater
than 64K (not verified). Change is to serialize the config object as a string and recreate
it when processing the job.

### CODE REVIEW

**Comment by ram-copart** [comment]
@velvia just a friendly reminder to review this PR

File: job-server/src/main/scala/spark/jobserver/JobManagerActor.scala [hunk 1]
```diff
@@ -41,6 +45,56 @@ object JobManagerActor {
   // Akka 2.2.x style actor props for actor creation
   def props(contextConfig: Config, daoActor: ActorRef): Props = Props(classOf[JobManagerActor],
     contextConfig, daoActor)
+
+  // Serializer StartJob message
+  class StartJobSerializer extends JSerializer {
```
Comment (velvia) [inline job-server/src/main/scala/spark/jobserver/JobManagerActor.scala]: I would move this class outside, perhaps to `spark.jobserver.util`

Comment (rmettu-rms) [inline job-server/src/main/scala/spark/jobserver/JobManagerActor.scala]: and add serialization test

**Comment by rmettu-rms** [comment]
@velvia made suggested changes, please review

**Comment by rmettu-rms** [comment]
@velvia friendly reminder to review the changes

**Pull request merged**